gno.land/cmd/gnoland/testdata/addpkg_namespace.txtar

```diff
@@ -0,0 +1,39 @@
+loadpkg gno.land/r/demo/users
+
+adduser gui
+
+gnoland start
+
+## test address namespace
+
+! gnokey maketx addpkg -pkgdir $WORK -pkgpath gno.land/r/$USER_ADDR_test1/one -gas-fee 1000000ugnot -gas-wanted 100000000 -broadcast -chainid=tendermint_test gui
+stderr 'unauthorized user'
+
+gnokey maketx addpkg -pkgdir $WORK -pkgpath gno.land/r/$USER_ADDR_gui/one -gas-fee 1000000ugnot -gas-wanted 100000000 -broadcast -chainid=tendermint_test gui
+stdout 'OK!'
+
+## test unregister namespace
+
+! gnokey maketx addpkg -pkgdir $WORK -pkgpath gno.land/r/guiland/one -gas-fee 1000000ugnot -gas-wanted 100000000 -broadcast -chainid=tendermint_test test1
+stderr 'unauthorized user'
+
+## test register namespace
+
+## XXX: THE FOLLOWING LINES REQUIRE BEING ABLE TO SET examples/gno.land/r/demo/users/users.admin=<test1_addr>
```

Comment:
let's do it.

Reply:
![Added via Giphy](https://media0.giphy.com/media/GcSqyYa2aF8dy/giphy.gif?cid=c9487743w4082aykxkyq5re3fjifmib6rn5nqoy5c4bsu0qf&ep=v1_gifs_search&rid=giphy.gif&ct=g)